test_team_chaos_coordinators.py

```diff
@@ -17,7 +17,35 @@ def test_crypto(self):
 
     def test_date_time(self):
         """Test DATE_TIME functionality"""
-
+        # positive Test case
+        prefix = '01'
+        middle = '01'
+        suffix = '2000'
+        test_str = prefix + '/' + middle + '/' + suffix
+        result = analyze_text(test_str,['DATE_TIME'] )
+        # expect result
+        self.assertGreater(len(result), 0, 'Result empty')
+        # check correct entity_type
+        self.assertEqual(result[0].entity_type, 'DATE_TIME')
+        # check the score
+        self.assertEqual(result[0].score, 0.6)
+
+        #context enhancement
+        # add context word
+        test_str += 'birthday'
+        result = analyze_text(test_str, ['DATE_TIME'])
+        # expect a result
+        self.assertGreater(len(result), 0, 'Result is Empty')
+        # check correct entity_type
+        self.assertEqual(result[0].entity_type, 'DATE_TIME')
+        # check the score
+        self.assertEqual(result[0].score, 0.44999999999999996)
```

Comment:
You are hitting a python precision problem here.  It's fine to do something like this
```
        # check the score
        self.assertEqual(round(result[0].score, 2), 0.45)
```